packages/key4hep-stack/common.py

```diff
@@ -15,7 +15,12 @@
 import spack.util.environment
 from spack.util.environment import *
 import spack.user_environment as uenv
-import spack.store
+
+try:
+    # https://github.com/spack/spack/pull/38944 renamed store -> STORE
+    from spack.store import STORE
+except ImportError:
+    from spack.store import store as STORE
```

Comment:
```suggestion
    from spack import store as STORE
```
However I'm going to make the change anyway because this is what was working before, and I haven't checked that database is accessible both from `spack.store` and `spack.store.store` for older versions